imap_processing/cli.py

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env python3
 # ruff: noqa: PLR0913
-"""Run the processing for a specific instrument & data level.
+"""
+Run the processing for a specific instrument & data level.
 
 This module serves as a command line utility to invoke the processing for
 a user-supplied instrument and data level.
 
-Use
----
+USES
```

Comment:
Should this be `Examples`? (to be consistent with how we are naming this in other places)